src/main/java/hudson/plugins/build_timeout/impl/DeadlineTimeOutStrategy.java

```diff
@@ -20,7 +20,7 @@
 /**
  * If the build reaches <tt>deadlineTime</tt>, it will be terminated.
  * 
- * @author Fernando Migu�lez Palomo <fernando.miguelez@gmail.com>
+ * @author Fernando Miguelez Palomo <fernando.miguelez@gmail.com>
```

Comment:
With the é in the name, I get encoding error when building. This happens since I changed the required jenkins core verion to 1.609.3. So I changed the é to e.